lib/stat_tracker.rb

```diff
@@ -69,4 +70,19 @@ def average_goals_by_season
       goals_by_season
     end
   end
+
+  def percentage_home_wins
+    home_wins = @game_collection.games.find_all {|game| game.home_goals > game.away_goals}
+    home_wins.count.to_f / (@game_collection.games.count.to_f).round(2)
```

Comment:
There is an all method that we need to refactor with for each collection.

Reply:
Yeah, I agree but I think we can do then when we get together and refactor it all together.

Reply:
sounds good.